app/upgrades.go

```diff
@@ -202,3 +208,16 @@ func IncreaseMaxGas(ctx sdk.Context, app *App) {
 	params.Block.MaxGas = 120_000_000
 	app.StoreConsensusParams(ctx, params)
 }
+
+func SetSanctionParams(ctx sdk.Context, app *App) error {
+	ctx.Logger().Info("Setting sanction params")
+	params := &sanction.Params{
+		ImmediateSanctionMinDeposit:   sdk.NewCoins(sdk.NewInt64Coin("nhash", 1_000_000_000_000_000)),
```

Comment:
Is this number in range for int64? We should try and use bigint everywhere given the recent issues the unit64 has exposed

Reply:
```
Max int64: 9_223_372_036_854_775_807
Our Value:     1_000_000_000_000_000
```
So we're within int64 bounds there.

Under the hood, `sdk.Coin` uses a `big.Int`. This `sdk.NewInt64Coin` function is just a wrapper for `sdk.NewCoin(denom, sdk.NewInt(amount))`.